### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by mehtamohit013**
## Describe your changes
When the PR is merged into master, CI is not running.
[Explanation](https://github.com/ploomber/jupysql/pull/600#issuecomment-1591679732)
## Issue number

Closes #X

## Checklist before requesting a review

- [ ] Performed a self-review of my code
- [ ] Formatted my code with [`pkgmt format`](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#linting-formatting)
- [ ] Added [tests](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#testing) (when necessary).
- [ ] Added [docstring](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#documenting-changes-and-new-features) documentation and update the [changelog](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#changelog) (when needed)

----
:books: Documentation preview :books:: https://jupysql--617.org.readthedocs.build/en/617/

### CODE REVIEW

**Comment by mehtamohit013** [comment]
I tested it out on ploomber dummy repository:

Case-1
- Created a new branch `CI`
- Pushed to that branch
- [Actions](https://github.com/ploomber/dummy/actions/runs/5270382135/jobs/9529803146#step:6:1)

Case-2
- Make a PR
- [Actions](https://github.com/ploomber/dummy/actions/runs/5270495162/jobs/9530076278?pr=52#step:6:1)

Working as intended

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -39,7 +39,15 @@ jobs:
...
         run: |
-          python -m pkgmt.fail_if_modified -b origin/master -e doc 
+          if [ "$GITHUB_EVENT_NAME" == "pull_request" ]
+          then
+            echo "Pull request, running check_doc"
+            python -m pkgmt.fail_if_modified -b origin/master -e doc 
+          else
+            echo "This is not a pull request event"
+            echo "Running all tests"
+            exit 1
```
Comment (tonykploomber) [inline .github/workflows/ci.yaml:49]: I guess `exit 1` will make the action failure?
Should we do `exit 0`?

[Ref](https://docs.github.com/en/actions/creating-actions/setting-exit-codes-for-actions#about-exit-codes)

Comment (mehtamohit013) [reply .github/workflows/ci.yaml:49]: To run all tests, we want `check_doc` to fail. The test main purpose to determine whether anything other than `doc` has been modified. The test will fail if anything other than `doc` was modified and will run all tests.

Comment (tonykploomber) [reply .github/workflows/ci.yaml:49]: OK just saw `continue-on-error: true`
So I guess that still marks the failing step as "succeeded"?

Comment (mehtamohit013) [reply .github/workflows/ci.yaml:49]: `continue-on-error: true` make sure that the jobs below will run irrespective of the failure or success of a step. 

I am using the statement the outcome of the step to determine whether the next test will run or not.
`if: needs.check_doc.outputs.check_doc_modified == 'failure'`

Comment (tonykploomber) [reply .github/workflows/ci.yaml:49]: I see, logically it looks good to me.
Need to merge to master to verify

**Review by tonykploomber — APPROVED** [review]

**Pull request merged**